tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_test.ts

```diff
@@ -322,4 +322,88 @@ describe('linked_time_fob_controller', () => {
       });
     });
   });
+
+  describe('typing step into fob', () => {
+    it('single time selection changed with fob typing', () => {
+      let fixture = createComponent({});
```

Comment:
It would be helpful to seed this with an actual initial linkedTime value since it is relevant to the test, instead of forcing the reader to scroll up to createComponent to see what the default value is.